core/src/test/java/com/linecorp/armeria/internal/server/annotation/AnnotatedServiceTest.java

```diff
@@ -773,6 +773,12 @@ public String customHeader5(@Header List<Integer> numbers,
                    String.join(":", strings);
         }
 
+        @Post("/headerNameSpecified")
+        public String headerNameSpecified(@Header("X-Line-Id") String id) {
```

Comment:
Can we use more peculiar casing? e.g. `X-x-FoO-bAr` ?

Reply:
Thanks updated. 😉